_posts/2020-03-02-newpipe-0.18.6-released.md

```diff
@@ -2,7 +2,7 @@
 layout: post
 title: "NewPipe 0.18.6 released: Hotfix for the last hotfix"
 short: "NewPipe 0.18.6 released"
-date: 2020-02-28 00:00:00 +0100
+date: 2020-03-02 00:00:00 +0100
```

Comment:
In a few minutes

Reply:
ahaha, I updated the file name, but not that date, classic.